charts/gardener/provider-local/templates/coredns/deployment.yaml

```diff
@@ -70,6 +70,8 @@ spec:
         securityContext:
           allowPrivilegeEscalation: false
           capabilities:
+            add:
+            - NET_BIND_SERVICE
```

Comment:
Please add a comment that this should be removed with `coredns:v1.12`.

Reply:
I've already added a reminder to the images.yaml. I think it is not necessary to duplicate this to every occurrence in the code.

Reply:
Personally, I prefer to tag all the locations that need to be changed, but I can also live with a single reminder in the images file.